scripts/generate_dump

```diff
@@ -523,24 +523,37 @@ save_bgp_neighbor() {
     local ns=$(get_vtysh_namespace $asic_id)
 
     neighbor_list_v4=$(${timeout_cmd} bash -c "vtysh $ns -c 'show ip bgp neighbors' | grep 'BGP neighbor is' | awk -F '[, ]' '{print \$4}' | awk /\\\./")
-    for word in $neighbor_list_v4; do
-        save_cmd "vtysh $ns -c \"show ip bgp neighbors $word advertised-routes\"" "ip.bgp.neighbor.$word.adv$asic_id"
-        save_cmd "vtysh $ns -c \"show ip bgp neighbors $word routes\"" "ip.bgp.neighbor.$word.rcv$asic_id"
-    done
+    if [ ! -z "$neighbor_list_v4" ]; then
+        v4_cmd="vtysh "
+        for word in $neighbor_list_v4; do
```

Comment:
Great optimization!
Is multiAsic case missed here?

Reply:
I think this API operates per ns. I would recommend to extend this once merged if we want to combine all ns into one.

Reply:
hi Sud, I think part of what FengPan might be thinking is that - do you mind trying this on the multi-asic case and make sure nothing regresses?

Reply:
@r12f  We don't have a multiasic platform. However we ran the sonic-mgmt show techsupport test and it passes with the changes in the PR https://github.com/sonic-net/sonic-mgmt/pull/15466